### PR TITLE
Support reuse of boilerplate files #110 

### PR DESCRIPTION
**Pull request by rachx**
Follows https://github.com/MarkBind/markbind/pull/130

### CODE REVIEW

File: lib/Page.js [hunk 1]
```diff
@@ -64,13 +64,19 @@ Page.prototype.prepareTemplateData = function () {
...
       .then(result => this.markbinder.resolveBaseUrl(result, {
         baseUrlMap: this.baseUrlMap,
         rootPath: this.rootPath,
       }))
       .then(result => fs.outputFileAsync(this.tempPath, result))
-      .then(() => this.markbinder.renderFile(this.tempPath))
+      .then(() => this.markbinder.renderFile(this.tempPath, {
+          baseUrlMap: this.baseUrlMap,
+          rootPath: this.rootPath,
+        }))
```
Comment (acjh) [inline lib/Page.js]: Indentation is *off*. Are you using eslint?

**Comment by rachx** [comment]
Should I create the _boilerplates folder by default in markbind init?

**Comment by acjh** [comment]
Not sure about creating an empty folder, but a Wiki page would be good.

**Comment by damithc** [comment]
> Should I create the _boilerplates folder by default in markbind init?

No harm doing so.

**Comment by rachx** [comment]
Updated https://github.com/MarkBind/markbind-cli/wiki/Including-contents

**Comment by acjh** [comment]
@Gisonrg Awaiting your review!

**Review by Gisonrg — APPROVED** [review]

**Pull request merged**